lms/djangoapps/courseware/features/common.py

```diff
@@ -18,6 +19,15 @@
 logger = getLogger(__name__)
 
 
+# pylint: disable=no-member
+@before.each_scenario
```

Comment:
Nit: If you inline the disable on this line (L23), it should only apply to that line, and you won't need to manually enable.  Same goes for putting L25 on the end of L26.